src/draw/dma2d/lv_draw_dma2d.c

```diff
@@ -82,19 +82,26 @@ void lv_draw_dma2d_init(void)
 #else
 #warning "LVGL can't enable the clock for DMA2D"
 #endif
-
     /* disable dead time */
     DMA2D->AMTCR = 0;
 
+#if defined(__ZEPHYR__)
+    IRQ_CONNECT(DMA2D_IRQn, 0, lv_draw_dma2d_transfer_complete_interrupt_handler, NULL, 0);
```

Comment:
IRQ_CONNECT wants the isr signature like this:
```
void isr_handler(const void *unused)
```

while the function has this signature:
```
void lv_draw_dma2d_transfer_complete_interrupt_handler(void);
```

Maybe we need a wrapper here?

Reply:
Wrapped, thanks for noticing @faxe1008